cypress/e2e/reports/activity-log/activity-log.spec.js

```diff
@@ -90,13 +90,13 @@ describe("Reports > Activity Log > Home", () => {
         );
       });
 
-      const from = moment()
+      const from = moment("2022-07-14")
```

Comment:
Why are we setting an initial value of july 14th 2022?

Reply:
This is because previously before this fix was applied, the initial date value was not passed to the datepicker, but now that we are passing the initial date value, we had to set the date for the test so that we can easily let cypress runner to pick the date on the datepicker, otherwise we had to figure out how many times cypress has to click the next month button in order to get to the current date.

Reply:
excellent. Please add that explanation as a comment. That is going to be important information if any revisits this test.